### DISCUSSION TITLE
AOSP 12.0 v400.f

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#2132] [General]
- January security patch level
- Uh, again, stuff I don't remember
- Fix speaker on Umidigi X and Gome U9 (Thanks @ponces )
- Fix DT2W and suspend-related feature on various Samsung devices

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v400.f'>AOSP 12.0 v400.f</a>.</em>

**Comment by RFX-LEGEND** [comment]
Any estimate on stable release?

**Comment by ghost** [comment]
There is no system-squeak-arm64-ab-gapps-secure.img.xz.
Please add that also!

**Comment by Skulking12** [comment]
Hi In this The media isn't transfer to wireless earbuds .
Calling sound Working on buds but Media Sound isn't working 😞

**Comment by tperraut** [comment]
Why this tag point to the same commit as v400.e ?

Thank you for your work
